### ISSUE TITLE
Some more enhancements of YASA

### ISSUE THREAD

**Issue by kezhenxu94** [yasa-org/yasa#25]
Here are some suggestions from @epugh

[ ] Links back to the legacy Solr admin, so if the users can’t do something in YASA, just pop me back to the legacy interface

[ ] Edit config files in Yasa

[ ] Streaming expressions support

[ ] More understanding of the shape of data, https://github.com/o19s/bearded-wookie

**Comment by MarcusSorealheis** [comment]
@kezhenxu94 and @epugh I think that as of today, it would seem that the priority is feature parity for that which we would like to keep in Solr. Shaping data is an entirely different effort and seems like it would be another plugin at this point. I'm not saying completely, but there is a lot of complexity there.

**Comment by kezhenxu94** [comment]
> @kezhenxu94 and @epugh I think that as of today, it would seem that the priority is feature parity for that which we would like to keep in Solr. 

Yes I agree totally, let’s just list the features (in individual issues) that are missing in this project but existed in the default admin and we could do that before this one

**Comment by epugh** [comment]
Thoughts on how to measure feature parity?  Is it at all useful to try and build an inventory of EVERYTHING solr admin does?   How much in Solr admin could be removed if we rethought what the admin tool does?  I thought about making a list of every feature that is in Solr Admin and comparing to YASA, but not sure how useful that would be...   Or maybe make a giant daunting list!

**Comment by kezhenxu94** [comment]
> I thought about making a list of every feature that is in Solr Admin and comparing to YASA, but not sure how useful that would be... Or maybe make a giant daunting list!

@epugh  both are good to me, I'm just wondering the diff feature list, and I'll align them ASAP, I need you buddies to help on that list

**Comment by kezhenxu94** [comment]
I rename the title of this issue to `Feature requests of YASA` as the checklist is new features compared to the old admin, let's list the diff in another issue, agree?

**Comment by epugh** [comment]
@MarcusSorealheis @kezhenxu94 I started a spreadsheet, just to see if this works:

https://docs.google.com/spreadsheets/d/13tjLXZezQmnRh2iSquj51a3VAUklUa_e6pVLLVu8sUM/edit?usp=sharing

What are your thoughts on this?  Does going through all of the Solr Admin and listing things at the level of detail that I did provide value?  I only did the main Dashboard, Logging, and Cloud aspects.   I could keep on going listing out all the areas, and then add a column that compares them to what is in Yasa????

**Comment by kezhenxu94** [comment]
> @MarcusSorealheis @kezhenxu94 I started a spreadsheet, just to see if this works:
> 
> https://docs.google.com/spreadsheets/d/13tjLXZezQmnRh2iSquj51a3VAUklUa_e6pVLLVu8sUM/edit?usp=sharing
> 

The list looks to be what I've been expecting

> What are your thoughts on this? Does going through all of the Solr Admin and listing things at the level of detail that I did provide value?

As we're going to reach the feature parity before Solr 9.0, this list gives a big picture (and the TODO list) before that, IMO, the list is valuable in this stage of YASA, and I'll develop YASA according to that list. Thanks @epugh very much

> I only did the main Dashboard, Logging, and Cloud aspects. I could keep on going listing out all the areas, and then add a column that compares them to what is in Yasa????

That would be much more appreciated so that I can find out the missing features in YASA and fill the gap

**Comment by epugh** [comment]
Great!   I'll keep plugging away at it ;-)   And try and update the "Already in YASA" column.   

I do worry that focusing too much on parity means we'll miss some opportunities to rethink what an Admin tool really should be!   Let's see how we do.

**Comment by epugh** [comment]
Hi @kezhenxu94 I hope your year is going well.   I've got some end of year time, if you want me to test out some more featurs to YASA ;-)

**Comment by kezhenxu94** [comment]
> Hi @kezhenxu94 I hope your year is going well. I've got some end of year time, if you want me to test out some more featurs to YASA ;-)

Thanks @epugh , hope you’re well. To be honest, there is no much progress recently because of my personal affairs. 😔 I’ll be back to this project when I’m done. I’ll ping you then. Thanks again. 

> Also, I’d like to see more people joining so that I don’t become the bottleneck of it
